spec/setup_acceptance_node.pp

```diff
@@ -23,22 +23,22 @@
 # $baseurl = "https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el${major}/x86_64/"
 $baseurl = "http://koji.katello.org/releases/yum/katello-nightly/pulpcore/el${major}/x86_64/"
 
-file { 'pulpcore-repo-extra-config':
-  path    => '/etc/yum/pulpcore.conf',
-  content => 'module_hotfixes=1',
-  before  => Yumrepo['pulpcore'],
-}
+$pulpcore_repo_conf = @("CONF")
+[pulpcore]
+baseurl=${baseurl}
+gpgcheck=0
+module_hotfixes=1
+CONF
 
-yumrepo { 'pulpcore':
-  baseurl  => $baseurl,
-  gpgcheck => 0,
-  include  => 'file:///etc/yum/pulpcore.conf',
+file { '/etc/yum.repos.d/pulpcore.repo':
+  ensure  => 'present',
+  content => $pulpcore_repo_conf,
 }
 
 # Needed as a workaround for idempotency
 if $facts['os']['selinux']['enabled'] {
   package { 'pulpcore-selinux':
-    ensure  => installed,
-    require => Yumrepo['pulpcore'],
+    ensure  => file,
```

Comment:
I think you meant to change the file resource, not the package one.

Reply:
.....I shouldn't push anything before I've had my coffee